src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -0,0 +1,418 @@
+module Algebra.Graph.Acyclic.AdjacencyMap (
+  -- * Data types and type aliases
+  PartialOrder,
+
+  -- * Data structure
```

Comment:
This section seems unnecessary. Also, I think `AdjacencyMap` should go first in documentation, as it is more important.